pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -646,10 +645,37 @@ func areMachineStatesEqual(ms1, ms2 state.NUMANodeMap) bool {
 				return false
 			}
 
-			if !reflect.DeepEqual(*memoryState1, *memoryState2) {
+			if memoryState1.TotalMemSize != memoryState2.TotalMemSize {
```

Comment:
Duplicate code below

```suggestion
			if memoryState1.TotalMemSize != memoryState2.TotalMemSize || memoryState1.SystemReserved != memoryState2.SystemReserved || memoryState1.Allocatable != memoryState2.Allocatable {
```

Reply:
done;